cfn-resources/resource-policy/README.md

```diff
@@ -1,13 +1,21 @@
 # MongoDB::Atlas::ResourcePolicy
 
-Congratulations on starting development!
+## Description
```

Comment:
I think you also need to update the schema with a documentationUrl similar to [project](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/project/mongodb-atlas-project.json#L200)

Reply:
Done!